### DISCUSSION TITLE
Customization options

### DISCUSSION THREAD

**Discussion by ellenbauer** [woocommerce/woocommerce#45049] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks?
I like that it will be easier to customize the cart and checkout experience via the editor.

## What do you think is missing from the Cart & Checkout blocks?
I hope we will get a lot more options. E.g. in the cart it is not possible to delete the product image or any other content element of the product. 
It would be helpful to have a background color option for the Cart total element, just for design purpose. 

As a theme author I think there are too many CSS styles added from the blocks. It makes it difficult to match the theme design and overwrites are currently still needed. Less CSS from the blocks would make a cleaner experience I believe.

**Comment by nielslange** [comment]
Thanks for this suggestion, @ellenbauer! I've added this suggestion to the discussion board, so that others can vote for this issue.

**Reply by ellenbauer** [reply]
Thank you @nielslange
